iceoryx_posh/source/roudi/memory/posix_shm_memory_provider.cpp

```diff
@@ -95,6 +93,13 @@ cxx::expected<void*, MemoryProviderError> PosixShmMemoryProvider::createMemory(c
         return cxx::error<MemoryProviderError>(MemoryProviderError::SIGACTION_CALL_FAILED);
     }
 
+    // create and map a shared memory region
```

Comment:
This should be on line 86 to be in between registering and unregistering the sigbus handler for having a custom error message when memory is not available.